src/cdomains/intDomain.ml

```diff
@@ -716,8 +716,8 @@ struct
   (* checks that x and y have the same range, and warns (in debug mode fails) if this is not the case *)
   let check_identical_range x y =
     if x <> y then
```

Comment:
According to your comments in #83 this should never happen, right?
Does it currently?
Maybe add a TODO comment to remove this and the option and just replace it by an assert once it's done.

Reply:
Ok, seems like it does https://travis-ci.com/github/goblint/analyzer/jobs/373279047

Reply:
Yes, I ran it on a few of the bigger programs and am looking at where it still happens there.
Ideally, we'd want to assert there, I agree.

Reply:
It now runs without any warnings for the `coreutils` and `drivers` test sets.

Reply:
Better keep this unresolved until the TODO is done.

Reply:
I removed the option and we now simply fail when this happens.